pkgs/development/python-modules/pycardano/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+# Python deps
+, blockfrost-python
+, cbor2
+, cose
+, ecpy
+, frozendict
+, frozenlist
+, mnemonic
+, poetry-core
+, pprintpp
+, pynacl
+, setuptools
+, typeguard
+, websocket-client
+}:
+
+let
+  cose_0_9_dev8 = cose.overridePythonAttrs (old: rec {
+    version = "0.9.dev8";
+    src = (old.src.override {
+      rev = "v${version}";
+      hash = "sha256-/jwq2C2nvHInsgPG4jZCr+XsvlUJdYewAkasrUPVaHM=";
+    });
+    pythonImportsCheck = [ "cose" ];
+  });
```

Comment:
Overriding for Python modules should be avoided by all costs and only used as a last resort. If the pinning `pycardano` for `cose` can't be relaxed then upstream has to fix it.